actors/runtime/src/util/downcast.rs

```diff
@@ -24,7 +24,7 @@ impl ActorDowncast for anyhow::Error {
         match downcast_util(self) {
             Ok(actor_error) => actor_error.wrap(msg),
             Err(other) => {
-                ActorError::new(default_exit_code, format!("{}: {}", msg.as_ref(), other))
+                ActorError::unchecked(default_exit_code, format!("{}: {}", msg.as_ref(), other))
```

Comment:
This downcasting smells iffy, but is used a _lot_.

Reply:
Yeah, the solution to this is: remove the usage of `anyhow` in all the ipld crates (amt,hamt, blockstore,encoding) using conrete error enums. `anyhow` is great as a "Final" error type, but not as a generic error from which you want to retrieve the original error again. (I have some wip code that does this, but it's a bit more work to finish)